src/api/integration-requests.md

```diff
@@ -54,8 +54,7 @@ Integration Requests enable terminal integrations and acceptance of asset types
   body ({
     accountId: 'Jaim1Cu1Q55uooxSens6yk',
     merchantId: '5ee0c486308f590260d9a07f',
-    type: 'epay',
-    terminalId: '11000021'
```

Comment:
Create Integration Requests endpoint doesn't need terminalId.

Reply:
@Neeraj-Ghodla "type" is mandatory though